node/src/accountant/mod.rs

```diff
@@ -883,42 +868,51 @@ impl Accountant {
         );
     }
 
-    fn handle_report_routing_service_consumed_message(
-        &mut self,
-        msg: ReportRoutingServiceConsumedMessage,
-    ) {
-        debug!(
-            self.logger,
-            "Accruing debt to wallet {} for consuming routing service {} bytes",
-            msg.earning_wallet,
-            msg.payload_size
-        );
-        self.record_service_consumed(
-            msg.service_rate,
-            msg.byte_rate,
-            msg.timestamp,
-            msg.payload_size,
-            &msg.earning_wallet,
-        );
+    fn msg_id(timestamp: SystemTime) -> String {
+        let microseconds_since_epoch = timestamp
+            .duration_since(UNIX_EPOCH)
+            .expect("time travelers")
+            .as_micros();
+        let lowest_3_digits = microseconds_since_epoch % 1000;
+        lowest_3_digits.to_string()
     }
 
-    fn handle_report_exit_service_consumed_message(
-        &mut self,
-        msg: ReportExitServiceConsumedMessage,
-    ) {
+    fn handle_report_services_consumed_message(&mut self, msg: ReportServicesConsumedMessage) {
+        let msg_id_opt = if self.logger.debug_enabled() {
```

Comment:
Would it make sense to put this check in Self::msg_id()? Maybe have it always return 0 if it fails? It looks from here like you're going to be doing this test everywhere you might call msg_id() anyway.